release notes/v0.50.0.md

```diff
@@ -6,8 +6,16 @@ k6 `v0.50.0` is here 🎉! This release includes:
 
 ## Breaking changes
 
-- `#pr`, `<small_break_1>`
-- `#pr`, `<small_break_2>`
+### Browser APIs to Async
+
+In future release we are going to be moving most of the browser APIs to asynchronous (promisfying them). We expect that this will affect majority of our users, which is why we are posting this upfront before we make the change itself. Here are the reasons for making this large breaking change:
```

Comment:
How about?


```suggestion
In future releases, we are going to be moving most of the browser APIs to asynchronous (promisifying them). We expect this will affect most of our users, so we are posting this upfront before making the change. Here are the reasons for making this large breaking change:
```